company/company.ru.robomarkup

```diff
@@ -11,85 +11,85 @@
 
-Surface weather must be much harsher than expected to have torn it apart like that, so no wonder you aren't picking up any survivors. Therefore I agree with your idea to try and use this robot you've found in your stead. Better safe than sorry as I can't lose ANY more of you guys this quarter.
+Погода на поверхности, должно быть, куда суровее, чем мы ожидали, раз уж так разодрало весь корабль. Неудивительно, что ты не нашел выживших. Поэтому я соглашусь с твоей идеей, попробуй использовать найденного робота. Лучше перебдеть, чем недобдеть, я не могу потерять еще кого-то из вас, ребята. Не в этом квартале.
 
-So get connected to your new best friend down there and find a way to salvage something from this. Because when we aren't making big hauls, I'm looking bad to Nigel. And if I'm looking bad, you're looking for a new job.
+Так что объединяйся с твоим новым лучшим другом там, внизу, и давайте, ищите что-нибудь полезное. Ведь если наш улов будет плох, я буду злиться на Найджела. А ты будешь злиться на меня и искать новую работу.
 
-Ciao for now,
-Judith
+Чао пока что,
+Джудит
 
 
 #!company id{c-underground}
-# Re: Iceni salvage report #6
+# Re: Отчет специалиста со Льдыни №6
 
-Finally some good news! From the looks of your scan, it's a subterranean entrance. They went underground! I don't think I've ever heard of this before.
+Наконец-то хорошие новости! Судя по снимкам, это вход в подземные помещения. Они ушли под землю! Не думаю, чтобы слышала о чем-то таком раньше.
 
-It means that the colonists WEREN'T all frozen to death/carried off into the sky by those icy winds. Rare metals, transmitters... even the people themselves could still be down there.
+Это означает, что не все поселенцы замерзли до смерти или были унесены в небо ледяным ураганом. Редкие металлы, передатчики... ну и сами люди могут быть все еще там.
 
-Can't wait to tell Nigel. Ooh... maybe it's an Iris thing. Or even a Martin thing! You find something juicy and I could be lunching on Titan with big M before the end of the fiscal year. Go team!
+Не могу дождаться, чтобы рассказать Найджелу! Ох... может, еще Айрис расскажу. Или даже Мартину! Ты найди там что-нибудь сочненькое, и я смогу обедать с большим М. на Титане еще до конца отчетного года. Вперед, команда!
 
-Sometimes I envy you, you know. Out there, thousands of miles from home with no distractions from family, friends or leisure activities. Completely alone with your thoughts. Many would kill for the kind of inner peace I imagine you experience.
+Иногда я завидую тебе, знаешь. Где-то там, в тысячах километров от дома, и ни семья, ни друзья, ни всякие очень приятные дела тебя не отвлекают. Наедине со своими мыслями. Многие бы убили за возможность найти тот внутренний покой, который тебе дарован.
 
-Keep up the adequate work. And don't accept any attitude from that robot.
+Продолжай в том же духе. И сильно не привязывайся к этому роботу.
```

Comment:
> don't accept any attitude from that robot.

This means don't take any back talk / disrespect from the robot, it's a bit of ludicrous management advice. Did that come accross in this translation?

Reply:
That's something I didn't get) I'll fix it.